src/main/java/com/avides/spring/rabbit/listener/container/DefaultMessageListenerContainer.java

```diff
@@ -37,6 +38,15 @@ public DefaultMessageListenerContainer(ConnectionFactory connectionFactory)
         setErrorHandler(new ValidationErrorHandler());
         setDefaultRequeueRejected(false);
         setPrefetchCount(500);
+        setMissingQueuesFatal(false);
+    }
+
+    @Override
+    protected BlockingQueueConsumer createBlockingQueueConsumer()
```

Comment:
Das Thema lässt sich nicht über einen IT testen, da man einen Cluster aufbauen müssten, bei dem man dann temporär verschiedene Ports blockiert. Ich habe deswegen mal den UT erweitert.. Das mit dem BlockingQueueConsumer lässt sich hierbei aber auch nicht testen, da dieser erst zur Laufzeit erstellt wird und man den danach per Reflection aus dem ListenerContainer ziehen müsste... Das ganze Thema wurde lokal durchgespielt.